### PR TITLE
Revert bad changes from merge forward PR

### PR DESCRIPTION
**Pull request by jkotalik**
Fixes https://github.com/dotnet/aspnetcore/pull/21630#issuecomment-628889794

cc @dougbu

### CODE REVIEW

File: .azure/pipelines/jobs/default-build.yml [hunk 1]
```diff
@@ -104,11 +104,16 @@ jobs:
...
+            # Visual Studio Build Tools
+            queue: BuildPool.Server.Amd64.VS2019.BT.Open
+          ${{ if eq(parameters.isTestingJob, true) }}:
+            # Visual Studio Enterprise - contains some stuff, like SQL Server and IIS Express, that we use for testing
+            queue: BuildPool.Server.Amd64.VS2019.Open
         ${{ if eq(variables['System.TeamProject'], 'internal') }}:
           name: NetCoreInternal-Pool
           # Visual Studio Enterprise - contains some stuff, like SQL Server and IIS Express, that we use for testing
-          queue: BuildPool.Windows.10.Amd64.VS2019.Pre
+          queue: BuildPool.Server.Amd64.VS2019
```
Comment (dougbu) [inline .azure/pipelines/jobs/default-build.yml:116]: Just confirmed /w a few Arcade folks. You can leave this file alone 😀 (I'm very very pleased we didn't break anything.)

**Review by dougbu — APPROVED** [review]
Narrow this two just eng/Version*.* and you're done.

**Comment by jkotalik** [comment]
@dougbu if you are in a hurry, feel free to merge this before validation to get https://github.com/dotnet/aspnetcore/pull/21630 moving.

**Comment by dougbu** [comment]
I'm _really_ starting to hate desktop `msbuild` ☹️

**Comment by dougbu** [comment]
If the Helix build succeeds, I'm going to ignore that OOM. But, I'm going to let the current build finish. (FYI Helix jobs don't get canceled when the pipeline that submitted them is canceled. So, canceling a build doesn't clean up as much as you'd like.)

**Comment by dougbu** [comment]
A fix we made to src/Framework/ref/Microsoft.AspNetCore.App.Ref.csproj in d99e7fecb0e6190d0838d3529372ce34d9fe7924 broke this build. Suggest we just leave things alone since #21630 is validating :ok: and we can undo the src/Framework/ref/Microsoft.AspNetCore.App.Ref.csproj change only after that goes in. #21630 also overwrites some of the versions in this PR and the next dependency update will make everything coherent again.

That is, if I'm correct, let's get #21630 in and close this PR.

**Comment by jkotalik** [comment]
I'm a little uncertain about the action plan then. So what you are saying is:
1. Close this PR.
2. Merge #21630 
3. Reopen another PR to revert the Microsoft.AspNetCore.App.Ref.csproj

Is that correct?

**Comment by dougbu** [comment]
@jkotalik yes you've got the plan down. #21630 is in (hurray 🚀) and the last bullet can happen whenever
> Reopen another PR to revert the Microsoft.AspNetCore.App.Ref.csproj

/fyi that change worked around an inconsistency between the incoming packages from dotnet/runtime that goes away now that we're targeting `net5.0` too.

**Pull request closed without merging**